js-exercises/animal-sort/animalSort.js

```diff
@@ -0,0 +1,17 @@
+const animalSort = animals => {
+  const animalList = animals.map(animal => animal);
```

Comment:
1. This line doesn't provide any use case since it will return the exact same array.

Reply:
Hi, as stated above, I am deep copying the array element, so that we do not mutate the original array.